pkgs/development/python-modules/bluepy-devices/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, bluepy
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "bluepy_devices";
```

Comment:
```suggestion
  pname = "bluepy-devices";
```

Reply:
This would revert your [previous request](https://github.com/NixOS/nixpkgs/pull/110399#pullrequestreview-573838141) to change the `pname`. Looks like that I misunderstood the comment.

The module name is `bluepy_devices`. Also, the name of source tarball starts with `bluepy_devices*` on PyPI. But it seems that PyPI is displaying the name acc. to PEP-0503 while `pip install` works with  `bluepy_devices` and  `bluepy-devices`.

Reply:
> This would revert your previous request to change the pname. Looks like that I misunderstood the comment.

I have talked to jonringer in the meantime and new packages should follow the pep (PEP503?) that converts _ to -.